audeer/core/utils.py

```diff
@@ -4,15 +4,20 @@
 from distutils.version import LooseVersion
 import functools
 import hashlib
+import importlib
 import inspect
 import multiprocessing
+import operator
+import pkg_resources
 import queue
 import subprocess
+import sys
 import threading
 import typing
 import uuid
 import warnings
 
+import audeer
```

Comment:
```suggestion
```

Reply:
I have no idea how this happened :) I removed it.